libs/workspace/src/builders/barista-build/utils/start-server.spec.ts

```diff
@@ -68,25 +68,30 @@ it('should capture an internal server error on route failure', async () => {
   expect.assertions(3);
 });
 
-it('should capture a child process crash', async () => {
-  try {
-    forkPid = (
-      await startServer(fakeServerPath, 3333, {
-        FAKE_ERROR: 'true',
-      })
-    ).pid;
-  } catch (error) {
-    forkPid = error.pid;
-    expect(error.error).toMatch(/Error: FAKED ERROR/);
-  } finally {
-    expect(typeof forkPid).toBe('number');
-    process.kill(forkPid);
-  }
+// Skip this test on windows, as it is flaky
+// TODO: Figure out on how to test this properly in the future.
+(process.platform === 'win32' ? it.skip : it)(
```

Comment:
It's a little sad that I needed to do this, but I was not able to figure out why this test was flaky on the CI. 
The error captured from the process was always a little different, but contained a path rather than the expected error string. 
The `finally` crashed afterwards, because the process was no longer available to be killed...

Reply:
It is ok – as long it is running on any os. The builder will always run on Linux for creating barista.
The windows checks are only for our contributors so that if they try to build or run things don't have broken environments